frontend/src/app/components/containers/complaints/outcomes/hwcr-prevention-education.tsx

```diff
@@ -75,7 +75,7 @@ export const HWCRComplaintPrevention: FC = () => {
   const preventionTypeList = useAppSelector(selectPreventionTypeCodeDropdown);
   const { personGuid } = useAppSelector(selectComplaintHeader(complaintType));
   const assigned = useAppSelector(selectComplaintAssignedBy);
-  const officerList = useAppSelector(selectOfficers);
+  const { officers } = useAppSelector((state) => state.officers);
 
```

Comment:
same concern as with the hwcr-complaint-assessment component, officers doesn't appear to be used